build.gradle.kts

```diff
@@ -46,6 +46,12 @@ allprojects {
     }
 }
 
+kotlin {
+    sourceSets.all {
+        languageSettings.optIn("kotlin.RequiresOptIn")
```

Comment:
what it does?

Reply:
It solves` This class can only be used with the compiler argument '-opt-in=kotlin.RequiresOptIn'` warnings. I tried add `freeCompilerArgs += "-opt-in=kotlin.RequiresOptIn"` ], but I got `e: Invalid argument: -opt-in kotlin.RequiresOptIn` error. [Here](https://youtrack.jetbrains.com/issue/KT-51708) I found this solution.

Reply:
the thing is that warning could be hidden by adding annotation or adding here compiler arg
please align the code / remove annotations in places where they are needed after adding this

Reply:
`"-opt-in=kotlin.RequiresOptIn"` is added after Kotlin 1.6.x and above, in lower versions still need to use `"-Xopt-in=kotlin.RequiresOptIn"`

Reply:
ok